foxy/distribution.yaml

```diff
@@ -6046,6 +6046,22 @@ repositories:
       url: https://github.com/SICKAG/sick_scan2.git
       version: master
     status: developed
+  simple_actions:
+    doc:
+      type: git
+      url: https://github.com/DLu/simple_actions.git
```

Comment:
it would be nice if the package.xml license were more specific, I think most people don't expect "BSD-2 clause" if the package.xml only says "BSD.

Reply:
This is the nitpickiest reason I've gotten for not merging into rosdistro.

Reply:
But what a pleasant surprise it will be when they find out they can claim David Lu endorses their work!

Reply:
> This is the nitpickiest reason I've gotten for not merging into rosdistro.

Sorry if that's annoying.
Reviewing rosdistro PRs is something I do almost automatically based on the review guidelines https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md.

One of the items to check for new packages added to a distro is if the License is open source and if it matches what the package.xml says. I found something strange and commented.